configs/common/modules_lmod.yaml

```diff
@@ -71,6 +71,7 @@ modules:
       - openssl
       - perl
       - pkgconf
+      - pkg-config
```

Comment:
Needed since pkg-config is built with gcc for the nvidia template, and our meta-modules logic doesn't allow loading this cleanly. Not needed as a module anyway.